### ISSUE TITLE
True path violations under NON proteinogenic amino acid metabolic process

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#28546]
- [x] arginine biosynthetic process via ornithine (GO:0042450) (NOW L-ARGININE)
- [ ] arginine catabolic process to ornithine (GO:0019547)
- [ ] L-methionine biosynthetic process from L-homoserine via cystathionine (GO:0019279)
- [ ] L-methionine biosynthetic process from homoserine via O-acetyl-L-homoserine and cystathionine (GO:0019280)
- [ ]  L-methionine biosynthetic process from homoserine via O-succinyl-L-homoserine and cystathionine (GO:0019281)

**Comment by pgaudet** [comment]
You mean they should be under 'proteinogenic'? If that's the case, you reported this 3 weeks ago here: 
https://github.com/geneontology/go-ontology/issues/28367

Should we close one as redundant?

**Comment by ValWood** [comment]
doh, in my head i was thinking this was different. I did know some were  not under proteinogenic that should be, but I didn 't realise they were actually in the wong place.

**Comment by deustp01** [comment]
Are we tripping here over the fact that some proteinogenic amino acid biosynthetic pathways include intermediates that are also amino acids, like ornithine, but that are not themselves proteinogenic, so depending on the molecule you look at the process is proteinogenic (arginine) or not (ornithine), which is why (I still think) imposing this distinction at the level of amino acid biosynthesis is dangerous. Impose it instead at the level of the processes that consume arginine or ornithine or glutamate ...

**Comment by pgaudet** [comment]
No, the problem is the ph/.3 forms, see the other ticket: https://github.com/geneontology/go-ontology/issues/28367#issuecomment-2203300882

**Comment by ValWood** [comment]
The ph forms is an issue, but I think @deustp01  is correct here. These have a weird parentage and it was on my list to report it.

For example 
L-methionine biosynthetic process from L-homoserine via cystathionine (GO:0019279)

has 
<img width="915" alt="Screenshot 2024-07-22 at 11 39 16" src="https://github.com/user-attachments/assets/03dee25e-d042-4913-a1e1-47f553707021">


Some children of  "methionine biosynthetic process" are descendants of the  associated intermediate metabolic process  (which has  a none proteinogenic amino acid parent )  This seems to be the case for all of these examples above.

**Comment by ValWood** [comment]
Isn't the problem  still going to exist  even after the zwitterion issue is resolved, because homoserine is an input ?
<img width="677" alt="Screenshot 2024-07-22 at 11 44 23" src="https://github.com/user-attachments/assets/e3952d8d-7b73-46b8-bf05-c6f31d8fefc0">

Is this the correct way to describe a pathway? I thought MFs have inputs and pathways have starts and ends?

**Comment by ValWood** [comment]
Still have the non-proteinogenic parentage

**Comment by edwong57** [comment]
I did a little digging, and L-homoserine is classified as a non-proteinogenic L-alpha-amino acid, so because the logical definiton of non-proteinogenic amino acid metabolic process is: 'metabolic process' and ('has primary input or output' some 'non-proteinogenic amino acid'), all of these terms will have the parent 'non-proteinogenic amino acid metabolic process.' We could take out the logical definition and then manually assert 'proteinogenic amino acid metabolic process.' Other suggestions?

**Comment by ValWood** [comment]
GO:0009092	homoserine metabolic process
is in the list of compounds I noted as 'pathway intermediates' as I worked through these GO-Cam pathways, so perhaps the solution is to obsolete that term?

I will need to check that because I only looked from a yeast perspective.

@pgaudet

**Comment by ValWood** [comment]
I'm not sure, it's at a branch point. I don't think we have clear rules yet...

Here is a possibly correct, but slightly more convoluted solution.
1) obsolete homoserine catabolic process (unused)
2) obsolete the unecessary grouping term homoserine metabolic  process because it is no longer required (single inheritance)
- Then, the methionine/ threonine biosynthesis terms could no longer have a "homoserine metabolic process" parent, and the logical def issue would be resolved.

**Comment by deustp01** [comment]
Hmmm. This seems like a lot of work to get around a distinction that artificially subdivides amino acid metabolism.

Now look at the parentage for [GO:0019273](https://www.ebi.ac.uk/QuickGO/term/GO:0019273) L-alanine biosynthetic process via ornithine and [GO:0019555](https://www.ebi.ac.uk/QuickGO/term/GO:0019555) L-glutamate catabolic process to ornithine.

**Comment by edwong57** [comment]
Hmm. Looks like there was a discussion in 2024 for logical definitions for metabolic processes - https://wiki.geneontology.org/Ontology_meeting_2024-04-29
was there a decision for specific guidelines?

**Comment by ValWood** [comment]
> This seems like a lot of work to get around a distinction that artificially subdivides amino acid metabolism.

It might be. But I think the current procedure is to remove the general metabolism term in the cases where only biosynthesis is relevant. In this case, the current procedure would be used, and would resolve the issue. So it wasn't just a workaround, but it also isn't quick to implement.

but @pgaudet  would need to confirm if this would be the right approach.

**Comment by deustp01** [comment]
One more rant on the original issue. Thinking about the recent obsoletion of MF terms like "proteolytic activity in connection with apoptosis", the attempt to classify amino acids according to their ultimate roles looks similarly precomposed and therefore in scope for a GO-CAM model but not for a MF or BP term.

**Comment by pgaudet** [comment]
> obsolete homoserine catabolic process (unused)
obsolete the unecessary grouping term homoserine metabolic process because it is no longer required (single inheritance)


This makes sense.

**Comment by edwong57** [comment]
I may be missing something, but I don't think that obsoleting these terms will fix the non-proteinogenic parentage. The  'homoserine metabolic process' is asserted, and when I remove it in Protege, 'non-proteinogenic amino acid metabolic process' is still a parent. Here's one explanation for why 
L-methionine biosynthetic process from L-homoserine via cystathionine (GO:0019279) has the parent 'non-proteinogenic amino acid metabolic process'

<img width="1451" height="312" alt="Image" src="https://github.com/user-attachments/assets/a27bd229-67d4-440c-aa49-17c86e0b979f" />

I'm not saying that these terms should *not* be obsoleted, I just don't think that doing so will fix the parentage issue.

**Comment by pgaudet** [comment]
You're right @edwong57 , just obsoleting the homoserine tersm will not address the incorrect non-proteinogenic parentage. 

I think we need to be more drastic and obsolete 
GO:0170039 proteinogenic amino acid **metabolic** process
GO:0170041  non-proteinogenic amino acid **metabolic** process

When I test this I dont get extra inferences anymore: 

<img width="1452" height="529" alt="Image" src="https://github.com/user-attachments/assets/f155185b-efab-4507-b6c2-ebff41031d0e" />

**Comment by ValWood** [comment]
@deustp01 commented
> the attempt to classify amino acids according to their ultimate roles looks similarly precomposed and therefore in scope for a GO-CAM model **but not for a MF or BP term.**

and maybe the groupings aren't necessary? As Peter says we will capture this via the causailty in the GO-CAMs?

**Comment by ValWood** [comment]
@sjm41  will your recent tickets make this one out of date?

**Comment by sjm41** [comment]
@ValWood Yes, we are systematically obsoleting (most of) the 'X amino acid biosynthesis/catabolic process via/from/to Y' variant pathways because they are problematic for several reasons (including the issues discussed here!) and because such variants are better represented as GO-CAMs. So I think you can close.

**Comment by ValWood** [comment]
Thanks!

**Comment by edwong57** [comment]
Closing as these are being taken care of in other tickets.
